frontend/src/components/stats/CirculatingSupplyStats.tsx

```diff
@@ -22,12 +22,12 @@ const CirculatingSupplyStats = ({ chartStyle }: Props) => {
           ({ date, circulatingTokensSupply, totalTokensSupply }) => ({
             date,
             circulatingTokensSupply: new BN(circulatingTokensSupply)
-              .divn(10 ** 6)
-              .divn(10 ** 6)
+              .div(new BN((10 ** 20).toString()))
+              .div(new BN((10 ** 4).toString()))
```

Comment:
Use `NEAR_NOMINATION` (it is BN number of 10**24): https://github.com/near/near-explorer/blob/master/frontend/src/components/nodes/NodesCard.tsx#L25